eng/Versions.props

```diff
@@ -169,7 +169,7 @@
     <RoslynOptProfRunSettingsGeneratorVersion>1.0.0-beta3.19057.1</RoslynOptProfRunSettingsGeneratorVersion>
     <RoslynToolsLightUpSystemRuntimeLoaderFixedVersion>4.3.0</RoslynToolsLightUpSystemRuntimeLoaderFixedVersion>
     <RoslynMicrosoftVisualStudioExtensionManagerVersion>0.0.4</RoslynMicrosoftVisualStudioExtensionManagerVersion>
-    <StreamJsonRpcVersion>1.3.23</StreamJsonRpcVersion>
+    <StreamJsonRpcVersion>1.5.43</StreamJsonRpcVersion>
```

Comment:
This one jumps out as a bit worrisome -- do you validate this version actually shipped in 16.0? If we're upgrading higher then it means we can't actually do servicing with this version if we had to.

Reply:
good point. Looks like we would be good with 1.5.38-beta.